### ISSUE TITLE
Materialized Views don't support schema migrations

### ISSUE THREAD

**Issue by malonehedges** [sqlc-dev/sqlc#1519]
### Version

1.12.0

### What happened?

The following migrations result in sqlc believing the materialized view exists in the main public schema, when it has in fact moved

I am running sqlc off the main branch.

```sql
-- migration 1:
CREATE MATERIALIZED VIEW something AS
select * from events
where event_type = 'sale'
order by created_at desc;


-- migration 2:
create schema computed_tables;
alter materialized view "something" set schema "computed_tables";


-- query:
-- name: SomeQuery :many
select id from "computed_tables"."something";
```

### Playground URL

https://play.sqlc.dev/p/7475c4ff0c2d81393cd0baa57ede84f0fa0af5bd7ef4885f68450adde9f01b17

### What operating system are you using?

macOS

### What database engines are you using?

PostgreSQL

### What type of code are you generating?

Go

**Comment by ludusrusso** [comment]
I can work on it if it's ok for @kyleconroy

**Comment by kyleconroy** [comment]
Fixed in v1.23.0
